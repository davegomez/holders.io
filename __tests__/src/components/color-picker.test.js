import React from 'react'
import ColorPicker from '../../../src/components/color-picker'
import renderer from 'react-test-renderer'
import { indetity } from '../test-utils'

test('ColorPicker component', () => {
  const component = renderer.create(
    <ColorPicker
      label='foo'
      color='#abc'
      onChangeComplete={ indetity }
    />
  )

  let tree = component.toJSON()
  expect(tree).toMatchSnapshot()
})
