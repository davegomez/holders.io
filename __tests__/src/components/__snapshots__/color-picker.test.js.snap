// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ColorPicker component 1`] = `
<label
  data-jsx={3085166267}
>
  foo
  <div
    className="picker-frame"
    data-jsx={3085166267}
  >
    <div
      className="picker"
      data-jsx={3085166267}
      onClick={[Function]}
      style={
        Object {
          "backgroundColor": "#abc",
        }
      }
    />
  </div>
</label>
`;
